templates/etc/td-agent/td-agent.conf

```diff
@@ -47,6 +47,9 @@
   # If a request is a mix of valid log entries and invalid ones, ingest the
   # valid ones and drop the invalid ones instead of dropping everything.
   partial_success true
+  # Do not split log entries with different log tags into different requests
+  # when talking to Stackdriver Logging API to avoid additional latency.
+  split_logs_by_tag false
```

Comment:
Why are we changing the default here and not in the plugin?

Reply:
Ah, right. Let me create a separate PR for that.